TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsTableViewManager.swift

```diff
@@ -7,7 +7,16 @@ class TradeItPortfolioAccountDetailsTableViewManager: NSObject, UITableViewDeleg
         case count
     }
 
+    private enum ACCOUNT_DETAIL_ROWS: Int {
+        case totalValue = 0
```

Comment:
Why do we assign a value only for `totalValue `? Should we assign all of them or none ?

Reply:
`enum: Int` auto-increments -- sometimes `1` is used as a starting value so I think it's idiomatic to define the first value.